### DISCUSSION TITLE
Advanced settings aren't taking effect

### DISCUSSION THREAD

**Discussion by smtilson** [sumatrapdfreader/sumatrapdf#4088] [General]
I have edited and saved the advanced settings file. They do not take effect. For example, I added a 195 zoom option and it does not show up. I have oopen and closed sumatra, I have restarted my machine, I have cleared the sumatra cache.

Any ideas why modifying the  settings file isn't taking effect?

**Comment by GitHubRulesOK** [comment]
custom zooms do not show in compiled application list so dont "show" in menu
but should be functioning as a step between the values either side you can check by stepping in and out and call CTRL+Y to see current value
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4d83e305-4072-46a2-832f-1634eeb0fa18)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2fbde05d-f468-45f6-bdc4-442909ef670f)

**Reply by smtilson** [reply]
Thank you very much. It does indeed work. I appreciate it.

However, the color settings have not had any impact at all. After I save the settings file is there something else I should do for it to take effect?

**Reply by GitHubRulesOK** [reply]
colours is to a degree work in progress as many have changed to emulate themes, Current pre-release should be the one to test

same file text set to green but inverted is not magenta because a theme is in force thus overrides to grey
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/29805d03-a420-4911-aa24-660fdaf42013)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/10d6f29f-fd34-4e35-9df6-110a5c88795d)
